docs/runbooks/VirtOperatorRESTErrorsBurst.md

```diff
@@ -0,0 +1,21 @@
+# VirtOperatorRESTErrorsBurst 
+
+## Meaning
+
+More than 80% of the rest calls in virt-operator pods failed in the last hour. 
+
+## Impact
+
+This alert suggests that the virt-operator is likely lost the connection to the apiserver. Cluster-level actions such as upgrading and controller reconciliation will likely not function. Customer workloads, i.e. VM and VMI, should not be affected.
+
+## Diagnosis
+
+Two common types of errors may lead to this alert:
+- The apiserver is overloaded and we run into timeouts. Issues like this can be identified by checking the apiserver metrics and looking at its response times, overall calls. Without customization, logs of KubeVirt apiserver pods are located on the master nodes, under the directory `/var/log/pods/*_kube-apiserver-*` and `/var/log/pods/*_virt-api-*`.
```

Comment:
@yuhaohaoyu Most of the time, users cannot access the master nodes directly. I believe we should change this section and recommend to fetch the logs with `kubectl logs` command. I can update if you are OK.

Reply:
@erkanerol Sorry for the delay. I have addressed this in two files. Thank.